pkg/client/client_test.go

```diff
@@ -157,6 +157,10 @@ func TestDependencyGraph(t *testing.T) {
 	)
 }
 
+func TestCyclicDependency(t *testing.T) {
```

Comment:
The test is empty and CI failed.

Reply:
yeah i am working on that test. this test wasn't supposed to be pushed, got committed with other changes on client_test.go
sorry about that

Reply:
@Peefy I have added the unit test, i couldn't understand why "windows test" ci is failing. 
![image](https://github.com/kcl-lang/kpm/assets/91385321/2f5bdf9b-27cf-4868-bc4c-4737619c6d3a)
any insights what went wrong ?

also e2e tests are failing as a cycle exist and now an error is displayed. I can fix the e2e test by changing test_suite.stdout files

Reply:
passed :+1: